schutzbot/containerbuild.sh

```diff
@@ -13,18 +13,22 @@ COMMIT=$(git rev-parse HEAD)
 IMG_TAGS=(
         "quay.io/osbuild/osbuild-composer:f32-${COMMIT}"
         "quay.io/osbuild/osbuild-composer:f33-${COMMIT}"
+        "quay.io/osbuild/osbuild-composer:ubi8-${COMMIT}"
 )
 IMG_PATHS=(
         "./containers/osbuild-composer/"
         "./containers/osbuild-composer/"
+        "./containers/osbuild-composer/"
 )
 IMG_FROMS=(
         "docker.io/library/fedora:32"
         "docker.io/library/fedora:33"
+        "registry.access.redhat.com/ubi8"
 )
 IMG_RPMREPOS=(
         "http://osbuild-composer-repos.s3-website.us-east-2.amazonaws.com/osbuild-composer/fedora-32/${ARCH}/${COMMIT}"
         "http://osbuild-composer-repos.s3-website.us-east-2.amazonaws.com/osbuild-composer/fedora-33/${ARCH}/${COMMIT}"
+        "http://osbuild-composer-repos.s3-website.us-east-2.amazonaws.com/osbuild-composer/rhel-8.3/${ARCH}/${COMMIT}"
```

Comment:
You use the `ubi8` container and tie it to `rhel-8.3`. Once `ubi8` updates to rhel-8.4, this will likely break.

Reply:
Hmm, I don't see an alternative as there are no minor releases of the UBI. That said, minor releases should be backwards compatible, and we don't depend on much (anything?) from the container that could break, I think, so I don't see how this would likely break. I posted an RFC PR last night that would make all this moot though, thoughts on that?

Reply:
Right. I thought I remembered ubi-images to having minor-release tags as well. But the information on any of this is quite sparse. If any `8.x` RPMs are meant to work on `ubi8` containers, then this should be fine.